gensrc/proto/internal_service.proto

```diff
@@ -621,5 +671,12 @@ service PInternalService {
     
     rpc update_fail_point_status(PUpdateFailPointStatusRequest) returns (PUpdateFailPointStatusResponse);
     rpc list_fail_point(PListFailPointRequest) returns (PListFailPointResponse);
+
     rpc exec_short_circuit(PExecShortCircuitRequest) returns (PExecShortCircuitResult);
+
+    rpc refresh_dictionary_cache_begin(PRefreshDictionaryCacheBeginRequest) returns (PRefreshDictionaryCacheBeginResult);
+    rpc refresh_dictionary_cache_commit(PRefreshDictionaryCacheCommitRequest) returns (PRefreshDictionaryCacheCommitResult);
+
+    rpc clear_dictionary_cache(starrocks.PClearDictionaryCacheRequest) returns (starrocks.PClearDictionaryCacheResult);
```

Comment:
can merge this to one RPC, like rpc process_dictionary_cache(request, cmd), as they use the same results only contains a status.

Reply:
fix it in next pr